example/lstm/README.txt

```diff
@@ -1,19 +1,33 @@
+## GOAL ##
+LSTM example using Platoon.
 
-To use the provided example, follow the steps below :
 
-1) Run the script lstm_master.py in a shell and leave it running until it
-   prints "Controller is ready".  This shouldn't take a long time.
+## CONTENT ##
+- README.txt
+- lstm_master.py
+- lstm_worker.py
+- imdb.py
 
-2) Run "python lstm.py init" until it prints "Param init done".
 
-3) At this point, you can run other workers (with lstm.py).  You may want to
-   run a single worker until it starts training to do all the compilation
-   required and not have all the workers fight for the compile lock.
+## HOW TO USE ##
+# USING THE LAUNCHER
```

Comment:
Right, I thought this was another `README.md` file.
It may be nice to make it a markdown file for a nicer display on GitHub in the future, but it's good for now.